lib/style.dart

```diff
@@ -0,0 +1,228 @@
+import 'package:flutter/material.dart';
+
+///This class represents all the available CSS attributes
+///for this package
+class Style {
+  ///CSS attribute "`background-color`"
+  Color backgroundColor;
+
+  ///CSS attribute "`color`"
+  Color color;
+
+  ///CSS attribute "`display`"
+  Display display;
+
+  ///CSS attribute "`font-family`"
+  String fontFamily;
+
+  ///CSS attribute "`font-size`"
+  double fontSize;
+
+  ///CSS attribute "`font-style`"
+  FontStyle fontStyle;
+
+  ///CSS attribute "`font-weight`"
+  FontWeight fontWeight;
+
+  ///CSS attribute "`height`"
+  double height;
+
+  ///CSS attribute "`list-style-type`"
+  ListStyleType listStyleType;
+
+  ///CSS attribute "`padding`"
+  EdgeInsets padding;
+
+  ///CSS attribute "`margin`"
+  EdgeInsets margin;
+
+  ///CSS attribute "`text-decoration`" -
+  TextDecoration textDecoration;
+
+  ///CSS attribute "`text-decoration-style`" -
+  TextDecorationStyle textDecorationStyle;
+
+  ///CSS attribute "`white-space`"
+  WhiteSpace whiteSpace;
+
+  ///CSS attribute "`width`"
+  double width;
+
+  //TODO modify these to match CSS styles
+  int baselineOffset;
+  String before;
+  String after;
+  TextDirection textDirection;
+  Border border;
+  Alignment alignment;
+  String markerContent;
+
+  Style({
+    this.backgroundColor,
+    this.color,
+    this.display,
+    this.fontFamily,
+    this.fontSize,
+    this.fontStyle,
+    this.fontWeight,
+    this.height,
+    this.listStyleType,
+    this.padding,
+    this.margin,
+    this.textDecoration,
+    this.textDecorationStyle,
+    this.whiteSpace,
+    this.width,
+    this.baselineOffset,
+    this.before,
+    this.after,
+    this.textDirection,
+    this.border,
+    this.alignment = Alignment.centerLeft, //TODO
```

Comment:
@dotneet, sorry for not responding to this earlier. I've created a constructor body that bases the alignment off of whether or not the container is using `display:block` or `display:list-item` so that custom rendered widgets can now be either a block element or an inline element (but defaulting to an inline element). I'll push the change to this branch today.